content/Education and Kits/Oplà IoT Kit/My-12-month-free-Create-plan-code-from-the-Opl-kit-is-not-working.md

```diff
@@ -8,18 +8,30 @@ The [Arduino Oplà IoT Kit](https://store.arduino.cc/opla-iot-kit) comes with 12
 
 ---
 
-1. Go to the [plan selection in the Arduino Digital Store](https://store.arduino.cc/digital/create#plans). Make sure the *Monthly* option is selected, and choose the Maker plan.
+1. Go to the [plan selection in the Arduino Digital Store](https://cloud.arduino.cc/plans).
 
-2. If you are not logged into your Arduino account, you will be taken to the login page. **Before proceeding, make sure you are logged into the account you want the code redeemed with.**
+2. In the _For individuals_ tab, select the **BILLED MONTHLY** option.
 
-3. Proceed with entering your billing information (no charges will be applied) until your order summary is displayed.
+   ![](img/opla-plan-select-monthly.png)
+
+3. In the Maker plan column, click PURCHASE.
+
+   ![](img/opla-plan-select-maker.png)
+
+4. If you are not logged into your Arduino account, you will be taken to the login page. **Before proceeding, make sure you are logged into the account you want the code redeemed with.**
+
+5. Add or update your billing information if needed, and click ADD PAYMENT INFORMATION.
+
+6. Click on **Add discount code**.
 
    ![Order summary before applying the discount code.](img/opla-plan-code-empty.png)
 
-4. In the **Discount Code** field, enter your discount code, and click on **APPLY CODE**. Your discount will be applied.
+7. Enter the code in the text field and click APPLY CODE if it is not automatically applied. Your discount will be applied in the purchase summary on the right.
```

Comment:
```suggestion
7. Enter the code in the text field and click APPLY CODE if it is not automatically applied. Your discount will be applied to the purchase summary on the right.
```

Reply:
It's a subtle distinction but I think the original is correct. The discount is applied as part of, or _within_ the summary, rather than the result of it. Google certainly favours this

* "applied in the order summary": 17,800 results
* "applied to the order summary": 5 results (although I find this somewhat difficult to believe)